_data/changelogs/component-date-picker.yml

```
@@ -2,6 +2,14 @@ title: Date picker
 type: component
 changelogURL:
 items:
+  - date: NNNN-NN-NN
+    summary: Improved keyboard navigation for screen readers.
+    summaryAdditional: Various screen readers now have more consistent and intuitive navigation using the respective table navigation controls.
```

What do you think about a little restructure? 

```suggestion
    summaryAdditional: This change enables table navigation controls in the calendar, which creates more consistent and intuitive navigation across browsers and screen readers. 
```

---

Lgtm! Resolved in [636c54e](https://github.com/uswds/uswds-site/pull/2218/commits/636c54e199f88e238eef9e323c4489853a15b296)